src/main/java/uk/gov/hmcts/reform/security/keyvault/KeyVaultService.java

```diff
@@ -49,43 +35,20 @@ public static KeyVaultService getInstance() {
     // Used by tests to inject the vault client
     KeyVaultService(KeyVaultClient vaultClient, LoadingCache<String, KeyBundle> keyByAliasCache, LoadingCache<String,
         KeyBundle> keyByIdentifierCache, LoadingCache<String, CertificateBundle> certificateByAliasCache) {
-        baseUrl = System.getProperty(BASE_URL_PROPERTY);
+        baseUrl = System.getProperty(KeyVaultConfig.VAULT_BASE_URL);
         this.vaultClient = vaultClient;
         this.keyByAliasCache = keyByAliasCache;
         this.keyByIdentifierCache = keyByIdentifierCache;
         this.certificateByAliasCache = certificateByAliasCache;
     }
 
     private KeyVaultService() {
-        baseUrl = System.getProperty(BASE_URL_PROPERTY);
-
-        String clientId = System.getProperty(CLIENT_ID_PROPERTY);
-        String clientSecret = System.getProperty(CLIENT_SECRET_PROPERTY);
-
-        KeyVaultCredentials keyVaultCredentials = new KeyVaultCredentials() {
-            @Override
-            public String doAuthenticate(String authorization, String resource, String scope) {
-                ExecutorService service = null;
-                try {
-                    service = Executors.newFixedThreadPool(1);
-                    AuthenticationContext context = new AuthenticationContext(authorization, false, service);
-
-                    ClientCredential credential = new ClientCredential(clientId, clientSecret);
-                    Future<AuthenticationResult> future = context.acquireToken(resource, credential, null);
-                    AuthenticationResult authenticationResult = future.get(30, TimeUnit.SECONDS);
-
-                    return authenticationResult.getAccessToken();
-                } catch (MalformedURLException | InterruptedException | ExecutionException | TimeoutException e) {
-                    throw new ProviderException(e);
-                } finally {
-                    if (service != null) {
-                        service.shutdown();
-                    }
-                }
-            }
-        };
+        baseUrl = System.getProperty(KeyVaultConfig.VAULT_BASE_URL);
```

Comment:
Why wouldn't you get the base url from the KeyVaultConfig?

Reply:
I'm not, it's just where the property name is defines